### DISCUSSION TITLE
Internal link creation via drag and drop does not work

### DISCUSSION THREAD

**Discussion by kicoln** [Zettlr/Zettlr#4556] [Q&A] [answered]
Thank you for providing nice software.

I am very troubled because I can not create internal links by drag and drop.
I'm currently using beta8.
This feature was working fine in 2.3.0.

The usage environment is as follows.
- Windows 10
- Intel 64bit

When I was using 2.3.0 I could drag and drop markdown files from the file tree to create internal links, or drag and drop image files in folders outside the workspace to create links. was
(I have "C:\Users\...\Documents\Notes" as my workspace and would often create internal links to images I saved to my desktop).

If I drag and drop from the file tree nothing happens (Zettlr logs nothing).
When I drag and drop an image file from another folder that I have opened in Windows Explorer, the notification "Could not open workspace (image name)" is displayed in the Windows notification column. Also, the Zettlr log shows the error "Could not open new root (absolute path of the image file)".

I don't know if it's a problem with my settings or a bug in the program, but please tell me what to do.

This question has been translated by Google Translate.
Sorry if it's hard to understand.

**Comment by nathanlesage** [comment, marked as answer]
Please open bug reports in such cases; I've found and fixed it, will come with the next nightly and 3.0.0 stable.

**Reply by kicoln** [reply]
Thank you for answering.
I'm glad you fixed it. I will make a bug report next time.
